pipelines/sqre/infra/build_sciplatlab.groovy

```diff
@@ -46,6 +51,10 @@ notify.wrap {
     }
 
     stage('build+push') {
+      def opts = ""
+      if jlbleed {
+              opts = "-e -s jlbleed"
```

Comment:
It is best practice to use `''` over `""` when string interpolation isn't being used.